src/vehicle_display.cpp

```diff
@@ -403,7 +403,9 @@ void vehicle::print_fuel_indicator( const catacurses::window &win, int y, int x,
             rate = consumption_per_hour( fuel_type, fuel_data->second );
             units = _( "mL" );
         } else if( fuel_type == itype_id( "battery" ) ) {
-            rate = power_to_energy_bat( total_epower_w(), 3600 );
+            int engine_and_parts_power = total_epower_w();
+            int reactor_power = total_reactor_epower_w();
+            rate = power_to_energy_bat( engine_and_parts_power + reactor_power, 3600 );
```

Comment:
```suggestion
            rate = power_to_energy_bat( total_epower_w() + total_reactor_epower_w(), 3600 );
```
No need to create separate variables for this.